lib/IRGen/GenReflection.cpp

```diff
@@ -790,11 +807,18 @@ void IRGenModule::emitBuiltinReflectionMetadata() {
   if (!IRGen.Opts.EnableReflectionBuiltins)
```

Comment:
You don't need this flag at all anymore then -- the original purpose was to enable emitting metadata for the various "special" builtins that are not directly referenced but still needed.